src/mig/modules/pkg/doc.rst

```diff
@@ -0,0 +1,68 @@
+====================================
+Mozilla InvestiGator: Package module
+====================================
+:Author: Aaron Meihm <ameihm@mozilla.com>
+
+.. sectnum::
+.. contents:: Table of Contents
+
+The package module (pkg) is intended to provide capabilities allowing
+inspection of software deployed on an agent system. The pkg module is
+primarily implemented through the use of the `mozoval <https://github.com/mozilla/mozoval>`_
+OVAL processing library.
+
+The module currently supports two functions:
+
+* Execution of OVAL definitions, performing vulnerability analysis functions
+* Matching against OS package managers, allowing inspection of deployed software versions
+
+Usage
+-----
+
+OVAL processing mode
+~~~~~~~~~~~~~~~~~~~~
+In OVAL processing mode, the module parses and executes an OVAL XML definition
+file supplied by the client. The OVAL definitions are transmitted to the agent
+in a compressed base64 encoded format. The agent then decodes this data, and
+pushes the OVAL definitions through the mozoval processor. The result of definitions
+are then returned to the investigator, indicating a true or false status.
+
+To use OVAL processing mode, the `oval` option is supplied on the command line,
+giving a properly formatted OVAL definition file as an argument.
+
+The `includefalse` parameter can be specified if false evaluations should be
+returned to the investigator. By default. only OVAL definitions that have
+evaluated to true are returned.
```

Comment:
This is a bit confusing. You may want to say that, by default, the module will only return packages that are deemed `vulnerable` based on the OVAL definitions, and by setting `includefalse`, the module will also return packages that are not vulnerable.

or something like that...